packages/block-editor/src/components/block-list/use-block-props/index.js

```diff
@@ -116,7 +113,6 @@ export function useBlockProps( props = {}, { __unstableIsHtml } = {} ) {
 		useBlockRefProvider( clientId ),
 		useFocusHandler( clientId ),
 		useEventHandlers( { clientId, isSelected } ),
-		useNavModeExit( clientId ),
 		useZoomOutModeExit( { editorMode } ),
```

Comment:
I wonder whether in future we'll want to normalize to using double click as a mechanism for existing this mode back to default mode? It's ok to remove for now though.